pkgs/development/python-modules/nevergrad/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, python
+, bayesian-optimization
+, cma
+, gym 
+, matplotlib
+, pandas
+, pytorch
+, pytest
+, typing-extensions
+, opencv4
```

Comment:
I usually like to alphabetically order my python package dependencies.

Reply:
Fixed. Thanks.